playwright/tests/orders.spec.ts

```diff
@@ -257,6 +257,10 @@ test("TC: SALEOR_84 Create draft order @e2e @draft", async () => {
 });
 
 test("TC: SALEOR_191 Refund products from the fully paid order @e2e @refunds", async () => {
+  // All steps of this test pass (including after hooks), but Playwright
+  // marks it as failed because of exceeding 30s timeout
+  test.slow();
```

Comment:
this marks test as slow, and triples the timeout, meaning something similar as if you were added 3x base timeout here. I does not look like fix for the flaky test.

Since you are fixing couple of test in single PR, please mark this as TODO, and leave ticket number for further fixing.

Reply:
Last time it failed the test threw the following error:

```
Test finished within timeout of 30000ms, but tearing down "context" ran out of time.
Please allow more time for the test, since teardown is attributed towards the test timeout budget.
```

here's the recording:
[32_timeout_PW.webm](https://github.com/user-attachments/assets/2464be05-9ab4-4cc1-b94d-65606c4240f4)